src/schema/markdown/serializer.js

```diff
@@ -0,0 +1,136 @@
+export const mention = (state, node) => {
```

Comment:
Separating serialiser logic as we could import only what's needed for the schema.